src/main.rs

```diff
@@ -366,30 +395,57 @@ fn get_args(matches: &ArgMatches) -> CliArgs {
             .collect(),
         None => Vec::new(),
     };
+    etcd_address_vec
+}
+
+/// Get scheduler port
+#[must_use]
+pub fn get_scheduler_port(matches: &ArgMatches) -> u16 {
```

Comment:
remove pub, also for other local functions

Reply:
done